mantidproject/mantid
# VSCode Guide Developer Documentation

**Description of work.**
A Developer guide on how best to work with VSCode aimed at new started mostly centred around operation on systems not operating on Windows.

**To test:**
- Build the dev-docs-html target
- Open "Build/Directory/dev-docs/html/index.html" in a browser
- Navigate down to Tools
- Click on VSCode
- Check that the page looks fine and the guide makes sense



*There is no associated issue.*

*This does not require release notes* as it is a developer doc

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
